### PR TITLE
Pages

### PR DESCRIPTION
**Pull request by irajbhinde**
Hey! Please review my code & let me know if any changes are to be made

Added - Home Page & Product Listing Page

GIFY -
home_plist_2

### CODE REVIEW

**Review by poojakelkar — COMMENTED** [review]
Looks Good to merge. Nice design

File: Get_Started/landing.css [hunk 1]
```diff
@@ -1,62 +1,87 @@
...
-    font-size: 1rem;
+
+.nav-bar {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 2rem 1rem;
+  padding-left: 2rem;
+  padding-right: 2rem;
```
Comment (mohitdhatrak) [inline Get_Started/landing.css]: you can just write `padding : 2rem;` instead of 3 lines

File: Pages/product_listing/product_listing.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+.container-two input {
+  width: 100%;
+}
+.container-two {
+  gap: 1rem;
+}
+.container-three,
+.container-four,
```
Comment (mohitdhatrak) [inline Pages/product_listing/product_listing.css:21]: add 1 line space  after each css block for neatness

**Review by mohitdhatrak — COMMENTED** [review]
Few suggestions given. Don't create PRs for 2 pages, it becomes too big, make separate for home and product. Now its fine, from next time you can try to do that

**Comment by irajbhinde** [comment]
> Few suggestions given. Don't create PRs for 2 pages, it becomes too big, make separate for home and product. Now its fine, from next time you can try to do that

Yeah, saw 18 files changed & realized that mistake, are you adding the images in git ignore for them to not show up in a PR ?

**Comment by mohitdhatrak** [comment]
> > Few suggestions given. Don't create PRs for 2 pages, it becomes too big, make separate for home and product. Now its fine, from next time you can try to do that
> 
> Yeah, saw 18 files changed & realized that mistake, are you adding the images in git ignore for them to not show up in a PR ?

No actually i am using 1 image as of now for all cards, anyways as only screens are required, would think about it later when i update the images! Also to add, more images is not the problem, the code was big too as you made product page as well as home page in 1 PR, I made 2 separate PRs that i wanted to say.

File: Pages/Home/home.html [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+              </div>
+              <div class="cards-title onimage">
+                <h4 class="strip">Digital Download</h4>
+              </div>
+            </div>
+          </div>
+        </a>
+        <a href="/Pages/product_listing/product_listing.html">
+          <div class="cards textovermedia">
+            <div class="img_text">
```
Comment (akash-1318) [inline Pages/Home/home.html:47]: for some classes you are using different naming convention , stick to one.

**Review by akash-1318 — COMMENTED** [review]
code looks really clean just one suggestion stick to one naming convention for naming css classes else everything looks good from my side.

**Pull request merged**